content/en/docs/tutorials/hello-minikube.md

```diff
@@ -55,10 +55,26 @@ If you installed minikube locally, run `minikube start`.
     minikube dashboard
     ```
 
+{{< note >}}
+This will enable the dashboard add-on, and open the proxy in the default web browser. You can create some Kubernetes resources on the dashboard, such as Deployment and Service.
+
+It's worth nothing that web browsers generally do not run properly as the root user, so if you are in an environment where you are running as root, see the session [Open Dashboard with URL](/docs/tutorials/hello-minikube#open-dashboard-with-url).
+
+To stop the proxy (leaves the dashboard running), abort the started process (`Ctrl+C`).
+{{< /note >}}
+
 3. Katacoda environment only: At the top of the terminal pane, click the plus sign, and then click **Select port to view on Host 1**.
 
 4. Katacoda environment only: Type `30000`, and then click **Display Port**.
 
+## Open Dashboard with URL
+
+If you don't want to open a web browser, the dashboard command can also simply emit a URL:
+
+    ```shell
```

Comment:
Would you remove the indentation of the command block (line 74)? See the preview page.